LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2014 Shippo (http://goshippo.com)
+Copyright (c) 2020 Shippo (http://goshippo.com)
```

Comment:
I don't know how licenses work but are we supposed to update every year?

Reply:
I believe we should be updating the copyright every year but a quick google search yields conflicting viewpoints - some say copyright should show the year of origination, some say copyright should be the most recent update date